packages/goto/package.json

```diff
@@ -29,10 +29,10 @@
   ],
   "dependencies": {
     "@browserless/devices": "^5.5.5",
-    "@cliqz/adblocker-puppeteer": "~0.12.0",
+    "@cliqz/adblocker-puppeteer": "~0.13.2",
     "debug": "~4.1.1",
     "debug-logfmt": "~1.0.2",
-    "got": "~9.6.0",
```

Comment:
could be possible keep `got`? I'm using it for the rest of the packages

Reply:
I can try to "polyfill" `fetch` using `got`. It just needs to be able to fetch text, JSON and array buffers. Will check it out.

Reply:
I think all those things (and more) are supported by `got`

Reply:
@Kikobeats I have made the change.